.gitignore

```diff
@@ -29,11 +31,14 @@
 /packages/lit-element/lit-element.*
 /packages/lit-element/lit.min.*
 /packages/lit-element/platform-support.*
+/packages/lit-element/hydrate-support.*
 
 # lit-html package
 /packages/lit-html/directives/
 /packages/lit-html/lit-html.*
 /packages/lit-html/parts.*
+/packages/lit-html/hydrate.*
+/packages/lit-html/private-ssr-support.*
```

Comment:
The `hydrate` module exports the `hydrate()` method used on the client-side for hydrating server-rendered lit-html templates. It's loaded on the _client_.

The `private-ssr-support` module re-exports the fields in `lit-html`'s `$private` export (with manled names) back to well-known symbol names for use in `lit-ssr`, so that `lit-ssr` is agnostic to whether the `lit-html` is in dev or prod mode. It's loaded by `lit-ssr` on the _server_.  It could just be called `private` maybe.  It's basically the "package protected symbols" that we want minified on the client, but used on the server.  Justin and I worked out this approach in the core lib chat last week.